src/pages/Call/index.tsx

```diff
@@ -129,9 +129,16 @@ const CallBase: React.FC<RouteComponentProps<TParams>> = ({ match }) => {
     [dispatch, callId]
   );
 
-  const leaveCallMemo = useCallback(
-    () => dispatch(push(`/feedback/${callId}`)),
-    [callId, dispatch]
+  const leaveCall = useCallback(
+    (exitType: CallExitType) => {
+      if (!call) {
+        dispatch(push("/"));
+        return;
+      }
+
+      dispatch(push(`/feedback/${call.id}?exitType=${exitType}`));
```

Comment:
the routing below makes me think this should be a path?

```suggestion
      dispatch(push(`/feedback/${call.id}/${exitType}`));
```